src/Chapter1.hs

```diff
@@ -632,7 +641,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = last1 (abs n) + last2 (abs n)
+  where
+    last1 :: Int -> Int
+    last1 x = mod x 10
+    last2 :: Int -> Int
+    last2 x = div (mod x 100 - last1 x) 10
```

Comment:
You don't need to subtract `last1 x` from `mod x 100`, because `div foo 10` removes the last digit anyway